docs/network-flow-visibility.md

```diff
@@ -43,6 +44,32 @@ For visualizing the network flows, Antrea monitors the flows in Linux conntrack
 module. These flows are converted to flow records, and then flow records are post-processed
 before they are sent to the configured external flow collector.
 
+## Getting Started
+
+To get started, let's ensure you have installed Antrea with the Flow Exporter
+feature enabled. You can refer to
```

Comment:
Could we give the default config here? I assume just enable the FlowAggregator feature gate (all other parameters with the default values should work) in antrea-agent conf?

I want not users to jump to another with lots of config options for the quick-start. But we can add links to Flow Exporter and Aggregator for users like to learn all configurations.